### PR TITLE
Update metadata for inject

### PR DESCRIPTION
**Pull request by idsvn**
SDBELGA-221: The metadata of the DPA ingest is not mapped correctly. 
'News Services' has to be 'NEWS'
'News Products' has to be 'GENERAL', 'ECONOMY', 'SPORTS' or 'CULTURE' as stated in https://dev.sourcefabric.org/browse/SDBELGA-151
'Credits' has to be 'DPA'
'Distribution' has to be 'default'
'City' has to be prefilled based on the original
'Country' has to be prefilled based on the original
'Editorial info' has to be empty 

SDBELGA-222.
'Credits' has to be 'AFP'
'Distribution' has to be 'default'
'Label' has to be empty

SDBELGA-223.
'Credits' has to be 'ANP'
'Distribution' has to be 'default'

update other  TASS, EFE
'Credits' has to be 'TASS','EFE'
'Distribution' has to be 'default'

Update unit-test

### CODE REVIEW

File: server/belga/io/feed_parsers/base_belga_newsml_1_2.py [hunk 1]
```diff
@@ -74,9 +74,10 @@ def parse(self, xml, provider=None):
                     if not [it for it in item.get('subject', []) if it.get('scheme') == 'news_products']:
                         product = {"name": 'GENERAL', "qcode": 'GENERAL', "scheme": "news_products"}
                         item.setdefault('subject', []).append(product)
-                    # add service is NEW
-                    service = {"name": 'NEWS', "qcode": 'NEWS', "scheme": "news_services"}
-                    item.setdefault('subject', []).append(service)
+                    item['subject'].extend([
```
Comment (ride90) [inline server/belga/io/feed_parsers/base_belga_newsml_1_2.py]: `item.setdefault('subject', []).extend(...)`

File: server/belga/io/feed_parsers/base_belga_newsml_1_2.py [hunk 2]
```diff
@@ -74,9 +74,10 @@ def parse(self, xml, provider=None):
...
+                        {"name": 'default', "qcode": 'default', "scheme": "distribution"},  # Distribution is default
+                        {"name": 'NEWS', "qcode": 'NEWS', "scheme": "news_services"},  # add service is NEW
```
Comment (ride90) [inline server/belga/io/feed_parsers/base_belga_newsml_1_2.py]: `{"name": 'NEWS', "qcode": 'NEWS', "scheme": "news_services"}` is it for all parsers or only for DPA?

File: server/belga/io/feed_parsers/belga_anpa.py [hunk 3]
```diff
@@ -61,15 +61,18 @@ def parse(self, file_path, provider=None):
                 b'([0-9]{1,2})-([0-9]{1,2}) ([0-9]{4})',
                 lines[1], flags=re.I)
             if m:
+                item['language'] = 'en'
+                item['priority'] = 2 if m.group(1).decode() == 'u' else 3
```
Comment (ride90) [inline server/belga/io/feed_parsers/belga_anpa.py]: Is there any ticket(s) regarding changes for this parser?

Comment (idsvn) [reply server/belga/io/feed_parsers/belga_anpa.py]: Yes, It like in file groovy, they use groovy file to mapping data in belga360.
you can check in https://dev.sourcefabric.org/browse/SDBELGA-151
file `kyodoText2newsml.groovy`

Comment (idsvn) [reply server/belga/io/feed_parsers/belga_anpa.py]: Belga's team tell me make the mapping like xsl, groovy file. The descriptions in this task is only a part of mapping, it use for guide.
note: description mapping AFP product via keywords that is wrong, in xsl it mapping with category.(they raise this bug)

File: server/belga/io/feed_parsers/base_belga_newsml_1_2.py (hunk 2 above)
Comment (idsvn) [reply server/belga/io/feed_parsers/base_belga_newsml_1_2.py]: Yes, it use for all for external source not Belga.

File: server/belga/io/feed_parsers/base_belga_newsml_1_2.py (hunk 1 above)
Comment (idsvn) [reply server/belga/io/feed_parsers/base_belga_newsml_1_2.py]: we don't use `setdefault` because it is sure a subjects( contain `product`) is exist.

**Pull request merged**